### ISSUE TITLE
var arg with n+1 arg help not working like 1-2 args

### ISSUE THREAD

**Issue by ldemailly** [grol-io/grol#224]
```
$ regexp
regexp(string, arg2..arg3) // returns true if regular expression matches the string (2nd arg)
```

should be
```
regexp(string, string, [bool])
```

compared with
```
$ json_go
json_go(any, [string]) // optional indent e.g json_go(m, "  ")
```
which is good

**Comment by ldemailly** [comment]
well the bug was min args was wrong, now it does
```
$ regexp
regexp(string, string, [boolean]) // returns true if regular expression matches the string (2nd arg)
```

**Comment by ldemailly** [comment]
fixed in the commit above in #240
